### PR TITLE
57 create cli wbaccinelli

### LINKED ISSUE #57: Create CLI for running the dashboard
Currently, the user has to clone the repo, navigate to the main folder and run `poetry run python eit_dash/main.py` to run the dashboard.
We want the users to be able to install the dashboard and run it with a simple command, especially when we will release this in pypi. This can be done by creating a small CLI with the run command available.

### PR DESCRIPTION
**Pull request by wbaccinelli**
Added CLI to run the dashboard. After installing the dashboard package with `pip install .`, the dashboard can be run using the command `eit-dash run`.

Closes #57

### CODE REVIEW

**Review by DaniBodor — CHANGES_REQUESTED** [review]

File: README.md [hunk 1]
```diff
@@ -17,6 +17,32 @@ this software.
...
+##### Install EIT Dashboard
+
+The first time that the dashboard is used, the repository needs to be cloned and the package has to be
+installed:
+
+```console
+git clone git@github.com:EIT-ALIVE/eit_dash.git
+cd eit_dash
+pip install .
+```
```
[the remark this answers is not in the document]
Comment (wbaccinelli) [reply README.md:29]: Good suggestion!

File: README.md [hunk 2]
```diff
@@ -17,6 +17,32 @@ this software.
...
+
+### 2. Running EIT Dashboard
+
+To run the installed dashboard the following command can be used:
+
+```console
+eit-dash run
```
[the remark this answers is not in the document]
Comment (wbaccinelli) [reply README.md:36]: Solved :)

File: README.md [hunk 3]
```diff
@@ -17,6 +17,32 @@ this software.
...
+```
+Open the resulting link in a browser (often something like `http://127.0.0.1:8050/`).
+Note that while the dashboard should work on any browser, if you are experiencing issues we recommend switching to
+Chrome or Firefox, as these are the browser where we do most of the testing.
```
[the remark this answers is not in the document]
Comment (wbaccinelli) [reply README.md:40]: It is in the user manual. I will add the sentence here as well.

[the remark this answers is not in the document]
Comment (wbaccinelli) [reply README.md:40]: I just realized that it is already [here](https://github.com/EIT-ALIVE/eit_dash/blob/796956db6d63d800fb79dd1b7405cc2fd896bf3b/README.md?plain=1#L6) in the README.md.

**Review by DaniBodor — APPROVED** [review]

**Pull request merged**